.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "2.7"
+install:
+    - pip install yapf==0.6.2
+script:
```

Comment:
I think that's only for strings. Validator still claims syntax error with "|" and ">". I looked through the other mlab repos and couldn't find any multiline commands and I couldn't find anything answering about non string multiline commands through just google searching.